lib/view_model/exchange/exchange_view_model.dart

```diff
@@ -1096,9 +1094,47 @@ abstract class ExchangeViewModelBase extends WalletChangeListenerViewModel with
 
   // Adding user's Erc20 tokens to the list of currencies
 
+  Future<Box<Erc20Token>> _openEvmTokensBoxFor(WalletInfo walletInfo) async {
```

Comment:
those are probably when solving a conflict, please remove them as they are already in TokenUtilities

Reply:
by those, I mean this function and the other ones for other chains